be/src/vec/data_types/serde/data_type_nullable_serde.cpp

```diff
@@ -238,13 +238,14 @@ void DataTypeNullableSerDe::write_one_cell_to_jsonb(const IColumn& column, Jsonb
                                                     Arena* mem_pool, int32_t col_id,
                                                     int row_num) const {
     auto& nullable_col = assert_cast<const ColumnNullable&>(column);
+    result.writeKey(col_id);
     if (nullable_col.is_null_at(row_num)) {
         // do not insert to jsonb
```

Comment:
remove this comment ，since null should be written

Reply:
done